earth_enterprise/src/third_party/gtest/SConscript

```diff
@@ -22,11 +22,13 @@ buildGtest = False
 
 # Check environment for libgtest.so
 conf = Configure(env)
-if not conf.CheckLib('gtest'):
-    buildGtest = True
-
 env = conf.Finish()
 
+# If building release, don't include gtest so scons won't add it as a dependency.
+if not env['release']:
+    if not conf.CheckLib('gtest'):
+        buildGtest = True
+
 # Build libgtest as a shared library. If we make gtest static it forces all
```

Comment:
I don't think you should need to build a library that's installed by a package.  The `gtest` package on Red Hat platforms already contains the built shared library.  Unfortunately, I think there isn't such a package on Ubuntu.

I'd, personally, rather build a Ubuntu package with the shared library like we did for RHEL 6, rather than have this SCons script.

Q: Are we packaging binaries in the RPMs that link against GoogleTest?  Do you know where the dependency comes from?

Reply:
Thanks, @calhoojp. I was wondering which binaries those were.  I could only imagine unit test executables being linked against `gtest`.

Reply:
That is all I can imagine, also. Which obviously doesn't make sense and isn't what we would want to be happening. Scons is completely new to me, so I'm not sure what exactly is causing this to happen. But I can do some more investigating and try to figure out what, exactly, is being linked.